File: sdkwork-backend-sdk-csharp/Models/AccountExchangeRateUpdateForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class AccountExchangeRateUpdateForm
    {
        public double? PointsToCashRate { get; set; }
        public string? Remarks { get; set; }
    }
}

File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/AccountExchangeRateUpdateForm.java
package com.sdkwork.backend.model;

public class AccountExchangeRateUpdateForm {
    private Double pointsToCashRate;
    private String remarks;

    public Double getPointsToCashRate() {
        return this.pointsToCashRate;
    }
    
    public void setPointsToCashRate(Double pointsToCashRate) {
        this.pointsToCashRate = pointsToCashRate;
    }

    public String getRemarks() {
        return this.remarks;
    }
    
    public void setRemarks(String remarks) {
        this.remarks = remarks;
    }
}
